features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/FlowDocument.java

```diff
@@ -724,4 +741,59 @@ public static FlowDocument from(final Flow flow) {
 
         return doc;
     }
+
+    public static EnrichedFlow buildEnrichedFlow(FlowDocument flowDocument) {
```

Comment:
I'd move this logic to the EnrichedFlow class instead of defining it here.

Reply:
Have added these back.